roles/openshift_hosted_logging/vars/main.yaml

```diff
@@ -1,6 +1,24 @@
-kh_kv: "KIBANA_HOSTNAME={{ openshift_hosted_logging_hostname | quote }}"
-es_cs_kv: "ES_CLUSTER_SIZE={{ openshift_hosted_logging_elasticsearch_cluster_size | string | quote }}"
-pmu_kv: "PUBLIC_MASTER_URL={{ openshift_hosted_logging_master_public_url | quote }}"
 ip_kv: "{{ 'IMAGE_PREFIX=' ~ target_registry | quote if target_registry is defined else '' }}"
```

Comment:
I don't know how widely used this role is so far, we haven't officially supported this yet so I'd prefer if we went ahead and changed `target_registry` to `openshift_hosted_logging_image_prefix` here.